### PR TITLE
Feature/custom namespacing

### PR DESCRIPTION
**Pull request by nedstankus**
Switched global name to $$PREBID_GLOBAL$$, replaced, by default, with 'pbjs'.
Updated references in /src, /test and /integrationExamples.

### CODE REVIEW

**Comment by nedstankus** [comment]
#115

**Comment by nedstankus** [comment]
@mkendall07 I resolved the conflicts with master and re-pushed.

**Comment by nedstankus** [comment]
I'll resolve conflicts again, as soon at #438 is resolved.

**Comment by nedstankus** [comment]
OK, @mkendall07 -- ready for review again.

**Comment by mkendall07** [comment]
@nedstankus 
Thanks for keeping this up to date and apologies on the delay in review. I've been cautious to accept this PR because of so many touch points. We have to do a full regression on all the bidder adapters because we don't have complete code coverage. 

With that said, i'll try to get this in by early next week.

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "type": "git",
     "url": "https://github.com/prebid/Prebid.js.git"
   },
+	"globalVarName": "pbjs",
```
Comment (mkendall07) [inline package.json]: nitpick: please soft tabs (spaces)

File: webpack.conf.js [hunk 2]
```diff
@@ -49,7 +49,21 @@ module.exports = {
             }
           ]
         })
-      }
+      },
```
Comment (mkendall07) [inline webpack.conf.js]: nit again: please use soft tabs

**Comment by mkendall07** [comment]
I'm going to merge this because it looks like it's working now. We'll monitor for any side effects as we move towards our next release iteration.

**Pull request merged**